.github/ISSUE_TEMPLATE/meeting-deliverables.md

```diff
@@ -7,36 +7,28 @@ title: "Deliverables Review Meeting: <YYYY-MM-DD>"
 
 # 2i2c Deliverables Review meeting
 
-This is a **30 minute** recurring meeting every two weeks on **Tuesday at 8:00AM Pacific time** (here's [**a timezone converter**](https://arewemeetingyet.com/Los%20Angeles/2000-01-01/08:00/2i2c%20Team%20Meeting#eyJ1cmwiOiJodHRwczovL2hhY2ttZC5pby9ZNVNCTXhWN1I2Q01xemVUWGdtNWtBIn0=), ignore the date!).
+This is a **30 minute** recurring meeting every **Tuesday at 8:00AM Pacific time** (here's [**a timezone converter**](https://arewemeetingyet.com/Los%20Angeles/2000-01-01/08:00/2i2c%20Team%20Meeting#eyJ1cmwiOiJodHRwczovL2hhY2ttZC5pby9ZNVNCTXhWN1I2Q01xemVUWGdtNWtBIn0=), ignore the date!).
 
 Our goal is to synchronize the team on the most important things to work on, and to divide work between one another so we know what to work on next.
 
 - **Video conference link:** https://bit.ly/zoom-holdgraf
 - **Calendar for future meetings:** [this Google Calendar](https://calendar.google.com/calendar/embed?src=c_4hjjouojd8psql9i1a8nd1uff4%40group.calendar.google.com&ctz=America%2FLos_Angeles)
+- **Meeting facilitator**: <INSERT FACILITATOR HERE>
+- **Meeting recorder**: <INSERT RECORDER HERE>
+- **Meeting timekeeper**: <INSERT TIMEKEEPER HERE>
 
 ## Meeting agenda
 
 _Follow the agenda below, checking boxes as we complete each step._
 
-### Activity Backlog
-_(5m) Quick review of [Activity Backlog](https://github.com/orgs/2i2c-org/projects/5?fullscreen=true) to let team members signal-boost if needed._
-
-- [ ] Team members signal-boost latest team sync items if they wish.
-- [ ] Note items that 2i2c team members wish to highlight for review.
-- [ ] Note important upcoming dates for the team.
-
-### Major deliverables
-
-_(20m) Overview major work items in our [Development Backlog](https://github.com/orgs/2i2c-org/projects/7?fullscreen=true)._
-
-- [ ] Discuss any high-priority items that need refinement.
-- [ ] Each 2i2c team member knows the next two things that they'll work on.
-- [ ] Each high-priority ready-to-work issue has at least one team member assigned to it.
-
-### Support steward transfer
-
-_(5m) Transition support steward roles to the next team member._
-
-- [ ] Preview support steward describes major support issues that have come up (with links to relevant issues)
-- [ ] Open issue for next Support Steward.
-- [ ] Close previous issue for Support Steward.
+- [ ] (5m) Quick review of [Activity Board](https://github.com/orgs/2i2c-org/projects/5?fullscreen=true) to track progress from last week.
```

Comment:
I'd like for us to make one board per week, so we can actually go through all of them from the last week. Helps make explicit decisions on what to continue working on, and what to drop / deprioritize.

Reply:
In general, I agree.
See my comment about that here: https://github.com/2i2c-org/team-compass/pull/193#discussion_r682934433

Reply:
Is the main benefit of "one board per week" so that we have a historical record of previous weeks? I can see benefit in that, though also worry that it is adding toil (somebody has to create the new board each week and close previous boards) as well as some complexity in our workflow since you'll have to remember a new URL each week for that week's board.

The alternative would be to do something like #191 to clean up the board at the end of each week.

I don't feel strongly about this though. If you really think "1 board per week" is the way to go, I'm fine trying that out.

Reply:
> Is the main benefit of "one board per week" so that we have a historical record of previous weeks?

The main benefit to me is the very tight scoping. There are 0 cards that can be ignored, there is a very clear reason each card is there. It also gives us a concrete way to look at cards that were not done the previous week, and make a clear decision on wether to carry it forward, descope, or what. It's ok to carry it forward (many tasks will take longer than a week!), but it becomes an explicit decision than something implicit.

The alternative to one board per week is to completely erase the board and start again. But one board per week is simpler (IMO). The process would be:

1. Before the meeting, create a new board
2. During the meeting, populate new board. A combination of carry over tasks from previous week and new cards
3. Archive current board in its state

Reply:
Yeah that makes sense - maybe I am just biasing things towards my own workflows? I like bookmarking things, so it is helpful for me to have "one URL to go to this week's activity board", and I don't have to keep re-updating the URL each week because it is now a new board. If others don't think this is inconvenient then I can live with it though.